website/docusaurus.config.js

```diff
@@ -134,8 +134,8 @@ var siteSettings = {
               to: '/guides/best-practices',
             },
             {
-              label: "Quickstarts",
-              to: "/quickstarts",
+              label: "Guides",
+              to: "/guides",
```

Comment:
Right above this nav item there is another `Guides` nav item which links to `/guides/best-practices`.  Thinking we'll have to rename this existing nav item.
![image](https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/748dd08a-d213-4b8a-8af4-5383ac968f61)

Reply:
Ope

Reply:
@runleonarun do you have a preference on what we want to rename/replace this with?

Reply:
@john-rock That should be "Best practices" for now and they should live in website/docs/best-practices/

Reply:
Got it thanks @runleonarun! That's updated now.